simulation/traffic_simulator/test/src/test_hdmap_utils.cpp

```diff
@@ -122,6 +122,116 @@ TEST(HdMapUtils, RoadShoulder)
   }
 }
 
+TEST(HdMapUtils, CanonicalizeNegative)
```

Comment:
Please describe the test target and expected result in comment.

Reply:
Comment added